install.sh

```diff
@@ -98,18 +98,30 @@ git clone -o cms-hh git@github.com:cms-hh/Resources.git HHStatAnalysis/Resources
 GITHUB_USER=$(git config user.github)
 if ! [ "x$GITHUB_USER" = "x" ] ; then
     cd HHStatAnalysis
-    git remote add $GITHUB_USER git@github.com:$GITHUB_USER/HHStatAnalysis.git
-    git fetch $GITHUB_USER
+    git ls-remote git@github.com:$GITHUB_USER/HHStatAnalysis.git &> /dev/null
```

Comment:
The purpose of this line is to check if the user's private fork of the repository exists. Even with option `--quiet`,  `git ls-remote` prints following error.

```
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Since it is totally ok if user doesn't have his private fork, I prefer to avoid such messages.